recipes/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.7" %}
+
+package:
+  name: pyedflib
+  version: {{ version }}
+
+source:
+  fn: pyEDFlib-{{ version }}.tar.gz
+  url: https://github.com/holgern/pyedflib/archive/v{{ version }}.tar.gz
+  sha256: b5626db80e7d2039160a54607bdac2c20842e80795d44df241a4478d36da7a1c
```

Comment:
Using the jinja templating variables is preferable for the `sha256`. See the example file in `recipes/example`